mentoring/tests/integration/test_dashboard.py

```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2014-2015 Harvard, edX & OpenCraft
```

Comment:
I still find having this in every module weird. Can't we put this stuff in a `LICENSE.md` file like every other project does?